src/corehost/cli/args.cpp

```diff
@@ -28,7 +28,7 @@ bool parse_arguments(
     // Get the full name of the application
     if (!pal::get_own_executable_path(&args.own_path) || !pal::realpath(&args.own_path))
     {
-        trace::error(_X("Failed to locate current executable"));
+        trace::error(_X("Failed to obtain full path to the executable using [%s]"), args.own_path.c_str());
```

Comment:
"using" doesn't really indicate why that path was used. How about `Failed to resolve the full path to the executable [%s]`